### PR TITLE
Update test_attribute_equals_hostname_constraint to support several big jobs

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: integration/tests/cook/test_basic.py [hunk 1]
```diff
@@ -1326,19 +1328,21 @@ def test_attribute_equals_hostname_constraint(self):
         # Based on the priority, canary should be scheduled ahead of big_job, but because
         # of the host-placement constraint they have to be scheduled on the same host.
         # The canary should be able to be scheduled, but big_job should never be scheduled.
-        big_job = util.minimal_job(group=group['uuid'],
+        big_jobs = [util.minimal_job(group=group['uuid'],
                                    priority=1,
                                    cpus=max_cpus)
```
Comment (dposada) [inline integration/tests/cook/test_basic.py]: Indentation seems off for these two lines with your diff

File: integration/tests/cook/test_basic.py [hunk 2]
```diff
@@ -1326,19 +1328,21 @@ def test_attribute_equals_hostname_constraint(self):
         # Based on the priority, canary should be scheduled ahead of big_job, but because
         # of the host-placement constraint they have to be scheduled on the same host.
         # The canary should be able to be scheduled, but big_job should never be scheduled.
-        big_job = util.minimal_job(group=group['uuid'],
+        big_jobs = [util.minimal_job(group=group['uuid'],
```
Comment (dposada) [inline integration/tests/cook/test_basic.py]: Maybe just `jobs` since the canary gets appended

File: integration/tests/cook/test_basic.py [hunk 3]
```diff
@@ -1326,19 +1328,21 @@ def test_attribute_equals_hostname_constraint(self):
...
                                    priority=1,
                                    cpus=max_cpus)
-        uuids, resp = util.submit_jobs(self.cook_url, [canary, big_job], groups=[group])
+                    for i in range(num_big_jobs)]
```
Comment (DaoWen) [inline integration/tests/cook/test_basic.py]: You're not actually using the `i`, right?
```python
for _ in range(num_big_jobs)]
```

File: integration/tests/cook/test_basic.py (hunk 2 above)
Comment (pschorf) [reply integration/tests/cook/test_basic.py]: Done

File: integration/tests/cook/test_basic.py (hunk 1 above)
Comment (pschorf) [reply integration/tests/cook/test_basic.py]: Done

File: integration/tests/cook/test_basic.py (hunk 3 above)
Comment (pschorf) [reply integration/tests/cook/test_basic.py]: Done

File: integration/tests/cook/test_basic.py [hunk 4]
```diff
@@ -1326,19 +1328,21 @@ def test_attribute_equals_hostname_constraint(self):
...
         # of the host-placement constraint they have to be scheduled on the same host.
         # The canary should be able to be scheduled, but big_job should never be scheduled.
-        big_job = util.minimal_job(group=group['uuid'],
-                                   priority=1,
-                                   cpus=max_cpus)
-        uuids, resp = util.submit_jobs(self.cook_url, [canary, big_job], groups=[group])
+        jobs = [util.minimal_job(group=group['uuid'],
+                                     priority=1,
+                                     cpus=max_cpus)
+                    for _ in range(num_big_jobs)]
```
Comment (dposada) [inline integration/tests/cook/test_basic.py]: Indentation still seems off here

Comment (pschorf) [reply integration/tests/cook/test_basic.py]: done

**Pull request merged**